src/jira/client/jira-client.ts

```diff
@@ -356,6 +356,14 @@ export const getJiraClient = async (
 				logger.info("Sending remoteLinks payload to jira.");
 				await instance.post("/rest/remotelinks/1.0/bulk", payload);
 			}
+		},
+		appProperties: {
+			create: (appKey: string, isConfiguredState: string) =>
+				instance.put(`/rest/atlassian-connect/1/addons/${appKey}/properties/isConfigured`, {
```

Comment:
Just by the PR me and K is doing, the url mostly will be [/atlassian-connect/1/addons-gh4j/properties/is-configured](https://stash.atlassian.com/projects/JIRACLOUD/repos/jira/pull-requests/80413/diff#jira-components/jira-plugins/atlassian-connect/core-components/core/src/main/java/com/atlassian/plugin/connect/plugin/rest/addons/GitHubForJiraAddonPropertiesResource.java)

Reply:
ignore me.